modules/core-services/services/application/appliation.go

```diff
@@ -769,11 +779,11 @@ func (a *Application) UpdatePublishItemRelations(request *apistructs.UpdateAppPu
 func (a *Application) PipelineCmsConfigRequest(request *apistructs.UpdateAppPublishItemRelationRequest) error {
 	for workspace, mk := range request.AKAIMap {
 		// bundle req
-		if err := a.bdl.CreateOrUpdatePipelineCmsNsConfigs(a.BuildItemMonitorPipelineCmsNs(request.AppID, workspace.String()),
-			apistructs.PipelineCmsUpdateConfigsRequest{
-				KVs: map[string]apistructs.PipelineCmsConfigValue{"AI": {Value: mk.AI},
-					"AK": {Value: mk.AK}},
-				PipelineSource: apistructs.PipelineSourceDice,
+		if _, err := a.cms.UpdateCmsNsConfigs(apis.WithInternalClientContext(context.Background(), "core-services"),
```

Comment:
what different between CreateOrUpdatePipelineCmsNsConfigs and UpdateCmsNsConfigs?

Reply:
It's the same. `UpdateCmsNsConfigs` is the original endpoint api name.